### ISSUE TITLE
Wistia Tutorial Links

### ISSUE THREAD

**Issue by kosiecki17** [LLK/scratch-gui#3510]
The Resources team would like to be able to create links from Wistia to the first card in each deck.

**Comment by kosiecki17** [comment]
Yes. We've noticed that a lot of kids do not click through from the video to the deck of cards and have created a "call to action" image in Wistia that we want to link to the first card.

**Comment by thisandagain** [comment]
@kosiecki17 Ok. We'll try to get this done, but it may slip into post-release.

**Comment by kosiecki17** [comment]
Thanks for following up on this.
